### PR TITLE
Import history of nginx

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10931,6 +10942,7 @@ geoemtry->geometry
 geogcountry->geocountry
 geographich->geographic
 geographicial->geographical
+geoio->geoip
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Rather specific, but probably generic enough to be of use.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11095,6 +11108,7 @@ grephic->graphic
 grestest->greatest
 greysacles->greyscales
 gridles->griddles
+grigorian->gregorian
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Please capitalise the correction.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -12698,6 +12713,7 @@ initialze->initialize
 initialzed->initialized
 initialzes->initializes
 initialzing->initializing
+initilialized->initialized
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Please add the BE with an `s`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -15336,6 +15356,7 @@ nmae->name
 nned->need
 nnumber->number
 no-overide->no-override
+nocachable->nocacheable
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This seems too specific.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -15336,6 +15356,7 @@ nmae->name
...
 nodel->model
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I know it's not yours but please also suggest nodal.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -15423,6 +15444,7 @@ nonte->note
 nontheless->nonetheless
 noo->no
 noone->no one
+nopsuh->nopush
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Too specific

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -20903,6 +20929,7 @@ smooting->smoothing
 smouth->smooth
 smouthness->smoothness
 smove->move
+smpt->smtp
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest `SMPTE`

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -4587,6 +4589,7 @@ combanations->combinations
 combatibility->compatibility
 combatible->compatible
 combiantion->combination
+combiations->combinations
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Please add the equivalent singular typo and correction.

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**